pubsub/kubemq/kubemq.go

```diff
@@ -0,0 +1,73 @@
+package kubemq
+
+import (
+	"context"
+	"github.com/dapr/components-contrib/pubsub"
+	"github.com/dapr/kit/logger"
+)
+
+type kubeMQ struct {
+	metadata         *metadata
+	logger           logger.Logger
+	ctx              context.Context
+	ctxCancel        context.CancelFunc
+	eventsClient     *kubeMQEvents
+	eventStoreClient *kubeMQEventStore
+}
+
+func NewKubeMQ(logger logger.Logger) pubsub.PubSub {
+	return &kubeMQ{
+		metadata:         nil,
```

Comment:
Nit: no need to initialize variables as `nil`.

Reply:
Changed as requested